
// let promise = new Promise((resolve, reject) => {
// 	console.log('Promise');
// 	resolve();
//   });
  
//   promise.then( () => {
// 	console.log('resolved.');
//   });

  const promise = new Promise(function(resolve, reject) {
	throw new Error('test');
  });
  promise.catch(function(error) {
	console.log(error);
  });
